test/policy-collection/policy_gatekeeper_operator_downstream_test.go

```diff
@@ -14,14 +14,20 @@ import (
 	policiesv1 "github.com/open-cluster-management/governance-policy-propagator/pkg/apis/policy/v1"
 	"github.com/open-cluster-management/governance-policy-propagator/test/utils"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 )
 
 var _ = Describe("RHACM4K-3055", func() {
+	var getComplianceState func(policyName string) func() interface{}
+
 	BeforeEach(func() {
 		if isOCP44() {
 			Skip("Skipping as this is ocp 4.4")
 		}
+
+		// Assign this here to avoid using nil pointers as arguments
+		getComplianceState = func(policyName string) func() interface{} {
+			return common.GetComplianceState(clientHubDynamic, userNamespace, policyName, clusterNamespace)
+		}
```

Comment:
Ok, in this case it's a little different because you're adding in some of the values you'd otherwise need to repeat... which is cool.